tests/completers/test_pip_completer.py

```diff
@@ -1,16 +1,22 @@
+import os
```

Comment:
Is this being used?

Reply:
And if it isn't, what happened to the flake8 check?

Reply:
Imports inside tests are not properly checked . I will check it.

Reply:
![image](https://user-images.githubusercontent.com/6702219/134620634-4910370c-05cd-4d16-a5ce-f4561e0b4e9a.png)

tests are excluded from checks. I will try to enable flake8 for them in another PR. I will remove os import in this.